src/rust/plugin-registry/migrations/20220202225101_create_plugin_deployment_table.sql

```diff
@@ -0,0 +1,19 @@
+-- Satisfies the "If success, mark plugin as being deployed in plugins table"
+-- requirement of RFC 0000 DeployPlugin.
+
+DO $$ BEGIN
+    IF to_regtype('plugin_deployment_status') IS NULL THEN
+        CREATE TYPE plugin_deployment_status AS ENUM ('fail', 'success');
+    END IF;
+END $$;
+
```

Comment:
Please do sweat the details on `plugin_deployment`. 
Should I use BIGSERIAL?
Should this also have a tenant-id or is just-plugin-id OK?

Reply:
plugin-id should be unique enough.. `BIGSERIAL` might not be a bad idea though just to be extra safe? I mean it feels unlikely we'll have 2.1 billion plugin deployments ever but who knows lol